src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.8.0'
```

Comment:
Please revert the change, we will bump version for these modules whose upcoming release section in changelog has some contents.